roles/irods_icat/tasks/setup_pam.yml

```diff
@@ -17,16 +17,6 @@
     state: absent
 
 
-- name: Ensure external user check is present
```

Comment:
Maybe replace with ansible.builtin.file with `state: absent` to clean up existing systems?